wgpu-core/Cargo.toml

```diff
@@ -106,6 +112,7 @@ rustc-hash = "1.1"
 serde = { version = "1", features = ["serde_derive"], optional = true }
 smallvec = "1"
 thiserror = "1"
+zerocopy = { version = "0.7", optional = true }
```

Comment:
I would prefer if we didn't take a dependency for this, or if we did it was bytemuck.